config/functions/functions_usage_summary.md

```diff
@@ -0,0 +1,548 @@
+Usage: IntSelect<SELECTION, Int1, Int2...>  
+SELECTION: FUNCTION  
+Returns SELECTION of N   
+If SELECTION is 0, the first integer is returned, if SELECTION is 1, the second and so forth.  
+N: numbers  
+return value: INTEGER  
+
+## [IsBetween](https://github.com/profezzorn/ProffieOS/blob/master/functions/isbetween.h)  
+Usage: IsBetween<F, BOTTOM, TOP>  
+Returns 0 or 32768 based F > BOTTOM and < TOP   
+F, BOTTOM, TOP: INTEGER  
+return value: INTEGER  
+
+## [IsLessThan](https://github.com/profezzorn/ProffieOS/blob/master/functions/islessthan.h)  
+Usage: IsLessThan<F, V>  
+Returns 0 or 32768 based on V  
+If F < V returns 32768, if F >= V returns 0   
+F, V: INTEGER  
+return value: INTEGER  
+
+## [LayerFunctions](https://github.com/profezzorn/ProffieOS/blob/master/functions/layer_functions.h)  
+Usage: LayerFunctions<F1, F2, ...>  
+F1, F2: FUNCTION  
+return value: FUNCTION  
+Returns (32768 - (32768 - F1) * (32768 * F2) / 32768)  
+This is the same as 1-(1-F1)*(1-F2), but multiplied by 32768.  
+Basically Mix<LayerFunctions<F1, F2>, A, B> is the same as Mix<F2, Mix<F1, A, B>, B>.  
+
+## [LinearSectionF](https://github.com/profezzorn/ProffieOS/blob/master/functions/linear_section.h)  
+Usage: LinearSectionF<POSITION, FRACTION>  
+POSITION: FUNCTION position on the blade, 0-32768  
+FRACTION: FUNCTION how much of the blade to light up, 0 = none  
+return value: FUNCTION  
+creates a "block" of pixels at POSITION taking up FRACTION of blade  
+
+## [MarbleF](https://github.com/profezzorn/ProffieOS/blob/master/functions/marble.h)  
+Usage: MarbleF<OFFSET, FRICTION, ACCELERATION, GRAVITY>  
+OFFSET: FUNCTION  0-32768, adjust until "down" represents is actually down  
+FRICTION: FUNCTION, higher values makes the marble slow down, usually a constant  
+ACCELERATION: FUNCTION, a function specifying how much speed to add to the marble  
+GRAVITY: FUNCTION higher values makes the marble heavier  
+return value: FUNCTION  0-32768, representing point on a circle  
+This is intended for a small ring of neopixels.  
+It runs a simulation of a marble trapped in a circular  
+track and returns the position of that marble.  
+Meant to be used with CircularSectionF to turn the marble  
+position into a lighted up section.  
+
+## [ModF](https://github.com/profezzorn/ProffieOS/blob/master/functions/mod.h)  
+Usage: ModF<F, MAX>  
+F: FUNCTION  
+MAX: FUNCTION (not zero)  
+When F is greater than MAX, F wraps to 0  
+When F is less than 0, F wraps to MAX  
+returns Integer  
+
+## [Mult, Percentage](https://github.com/profezzorn/ProffieOS/blob/master/functions/mult.h)  
+Usage: Mult<F, V>  
+Fixed point multiplication of values F * V,   
+fixed point 16.15 arithmetic (32768 = 1.0)  
+(2*2 would not result in 4),   
+(16384 * 16384 = 8192, representation of 0.5*0.5=0.25)   
+most blade functions use this method of fixed point calculations  
+F, V: INTEGER,   
+return value: INTEGER  
+
+Usage: Percentage<F, V>  
```

Comment:
It would be nice if Percentage could have it's own header here.

Reply:
done.